docker/logical-backup/Dockerfile

```diff
@@ -15,6 +15,8 @@ RUN apt-get update     \
         gnupg \
         gcc \
         libffi-dev \
+        nodejs \
+        npm \
```

Comment:
hm ins't there another CLI we can use to avoid these dependencies? What about [az](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-cli)?